### DISCUSSION TITLE
NPM Package Feature Request

### DISCUSSION THREAD

**Discussion by FelixWaweru** [FelixWaweru/elevenlabs-node#27] [General]
Hey everyone 👋
Glad to see the community growing and we hope you're finding this package useful to your projects.

As we work on adding more features to the package, what features you would like to see added to the package in future updates?

These features can either be from the existing [ElevenLabs API](https://api.elevenlabs.io/docs) or new functionality that we can add to the NPM package.

**Comment by werneroi** [comment]
Hi. First of all thank you for this package! helps me a lot!!!. I wonder if it is possible to also add the style and the use_speaker_boost to the parameters of the textToSpeech?
Thanks!

**Reply by FelixWaweru** [reply]
Hey @werneroi 
That's awesome! I'm glad you like the package :)

Thanks for the suggestion. This sounds like a great feature to have in the package so I'll add this to our release schedule. 🚀
